helics/capi.py

```diff
@@ -8407,6 +8459,20 @@ def helicsInputGetBytes(ipt: HelicsInput) -> bytes:
         return ffi.unpack(data, length=actualSize[0])
 
 
+def helicsInputGetStringSize(ipt: HelicsInput) -> HelicsDataBuffer:
```

Comment:
this doesn't seem right?   Is the name wrong or copy past error?

Reply:
Copy paste error.